src/templates/BlogPost.js

```diff
@@ -80,6 +81,10 @@ export const pageQuery = graphql`
     site {
       siteMetadata {
         title
+        demoUrl
```

Comment:
where does this info come from?

Reply:
https://github.com/RoadieHQ/marketing-site/blob/master/gatsby-config.js#L7

Reply:
thanks